src/components/Signin.jsx

```diff
@@ -0,0 +1,215 @@
+import React,{useState} from 'react';
+import undraw_login from '../components/assets/images/undraw_Login.svg';
+import  '../components/assets/css/Signin.css';
+import styled from 'styled-components';
+
+const SocialMediaLinks = styled.ul`
+  display: inline;
+  list-style:none;
+`;
+
+const Content = styled.div`
+  width: 70%;
+`;
+const Authenticate =styled.button`
+  width:30%;
+`;
+
+const ForgotPasswordButton = styled.button`
+  width:50%;
+  margin-left:5px;
+`;
+
+const OR = styled.div`
+  margin:40px;
+`;
+
+const LinkedinLink = styled.a`
+  justify-content:space-around;
+`;
+
+const LinkedinLinkContent = styled.p`
+  fontsize:18;
+  text-align:center;
+`;
+
+const LinkedinIcon = styled.i`
+  font-size:32px;
+  color:blue;
+  float:left;
+  background-color:white;
+`;
+
+
+const Signin = () => {
+    //const [status, setStatus] = useState("");
+    const [formData, setFormData] = useState({
+        name: '',
+        email: '',
+        password:''
+    });
+
+    const {name, email, password} = formData;
+
+    const onChange = (e) => {
+        e.persist();
+        setFormData((prevState) => {
+            return {
+                ...prevState,
+                [e.target.name]: e.target.value,
+            };
+        });
+    };
+
+    const submitForm = async (ev) => {
+        // ev.preventDefault();
+        // const config = {
+        //     headers: {'Accept': 'application/json'}
+        // }
+        // const res = await axios.post('https://formspree.io/xzbjwkae', formData, config);
+        // if (res.status === 200) {
+        //     setFormData({name: '', message: '', subject: '', email: ''});
+        //     setStatus("SUCCESS");
+        // } else {
+        //     setStatus("ERROR");
+        // }
+    }
+  return (
+    <div>
+      <h1 className="text-center">Signin</h1>
+      <Content
+                className="container-fluid rounded shadow-lg py-4 px-3 mt-5"
+            >
+                <div className="row mt-2">
+                    <div className="col-md-6">
+                        <img
+                            src={undraw_login}
+                            alt="undraw_login"
+                            className="float-left"
+                            style={{width: "100%", height: "100%"}}
```

Comment:
You can use bootstrap class for this. Don't use Inline CSS.

Reply:
bro as you can read the comment in the next line,  i tried to remove and replace but didnt work.